src/ui/Icons/Icon.tsx

```diff
@@ -760,13 +760,27 @@ export const findByName = function (name?: string): IconType | undefined {
   return icon;
 };
 
+const iconColorsClasssNamesMap = {
+  "error:red": "fill-red-500",
+  "error:orange": "fill-orange-500",
+  "error:warning": "fill-orange-500",
+  "warning:orange": "fill-orange-500",
```

Comment:
error and warning were just example icons i.e. the name of the icons should be split by a single `:` before the first part is used to find the icon and the second part to apply a style/color